### PR TITLE
Protobuf, Glog fixes

### PR DESCRIPTION
**Pull request by Arfrever**
(CC @dlan17)

### CODE REVIEW

**Comment by SoapGentoo** [comment]
as discussed, please remove full-stops. No arguments.

File: dev-python/protobuf-python/protobuf-python-9999.ebuild
Comment (thesamesam) [inline dev-python/protobuf-python/protobuf-python-9999.ebuild:52]:
```suggestion
	DISTUTILS_ARGS=( --cpp_implementation )
```

File: dev-python/protobuf-python/protobuf-python-9999.ebuild
Comment (thesamesam) [inline dev-python/protobuf-python/protobuf-python-9999.ebuild:4]:
```suggestion
EAPI="8"

```

File: dev-python/protobuf-python/protobuf-python-3.19.4.ebuild
Comment (thesamesam) [inline dev-python/protobuf-python/protobuf-python-3.19.4.ebuild:27]:
```suggestion
```

File: dev-python/protobuf-python/protobuf-python-3.19.4.ebuild
Comment (thesamesam) [inline dev-python/protobuf-python/protobuf-python-3.19.4.ebuild:19]: Please drop this branch.

File: dev-python/protobuf-python/protobuf-python-3.19.4.ebuild
Comment (thesamesam) [inline dev-python/protobuf-python/protobuf-python-3.19.4.ebuild:13]: Why?

File: dev-python/protobuf-python/protobuf-python-3.19.4.ebuild
Comment (thesamesam) [inline dev-python/protobuf-python/protobuf-python-3.19.4.ebuild:4]:
```suggestion
EAPI="8"

```

File: dev-python/protobuf-python/protobuf-python-3.19.4.ebuild
Comment (thesamesam) [inline dev-python/protobuf-python/protobuf-python-3.19.4.ebuild:7]: Does PEP517 work?

File: dev-libs/protobuf/protobuf-3.19.4.ebuild
Comment (thesamesam) [inline dev-libs/protobuf/protobuf-3.19.4.ebuild:133]:
```suggestion
		DOCS+=( examples )
```

File: dev-libs/protobuf/protobuf-3.19.4.ebuild
Comment (thesamesam) [inline dev-libs/protobuf/protobuf-3.19.4.ebuild:92]:
```suggestion
		options+=( --with-protoc="$(pwd)/src/protoc" )
```

File: dev-libs/protobuf/protobuf-3.19.4.ebuild
Comment (thesamesam) [inline dev-libs/protobuf/protobuf-3.19.4.ebuild:75]: You may be able to do this unconditonally.

File: dev-libs/protobuf/protobuf-3.19.4.ebuild
Comment (thesamesam) [inline dev-libs/protobuf/protobuf-3.19.4.ebuild:41]:
```suggestion
DOCS=( CHANGES.txt CONTRIBUTORS.txt README.md )
```

**Review by thesamesam — CHANGES_REQUESTED** [review]

**Review by SoapGentoo — CHANGES_REQUESTED** [review]
@dlan17 please only merge this PR once **everything** has been fixed. Unfortunately, in previous PRs, Arfrever has snuck in downgrades contrary to the proxy-maintainer's wishes (https://github.com/gentoo/gentoo/pull/24036#discussion_r797432729).

File: dev-python/protobuf-python/protobuf-python-3.19.4.ebuild
Comment (Arfrever) [reply dev-python/protobuf-python/protobuf-python-3.19.4.ebuild:13]: Otherwise `git-r3.eclass` would download (into `${DISTDIR}/git3-src`) copies of unused repositories which are listed in `.gitmodules` file:
https://github.com/protocolbuffers/protobuf/blob/master/.gitmodules:
```
[submodule "third_party/benchmark"]
	path = third_party/benchmark
	url = https://github.com/google/benchmark.git
[submodule "third_party/googletest"]
	path = third_party/googletest
	url = https://github.com/google/googletest.git
	ignore = dirty
```
`dev-java/protobuf-java` and `dev-python/protobuf-python` would never use them, while C++ `dev-libs/protobuf` is patched to use system version of `dev-cpp/gtest` (and it does not use `dev-cpp/benchmark`).

File: dev-python/protobuf-python/protobuf-python-3.19.4.ebuild
Comment (Arfrever) [reply dev-python/protobuf-python/protobuf-python-3.19.4.ebuild:7]: No:
```
 *   DISTUTILS_ARGS are not supported in PEP-517 mode
```
Ebuild has `DISTUTILS_ARGS=(--cpp_implementation)`.
Passing `--cpp_implementation` is required to enable C++ implementation.
[setup.py](https://github.com/protocolbuffers/protobuf/blob/v3.19.4/python/setup.py#L188-L198) contains:
```
def get_option_from_sys_argv(option_str):
  if option_str in sys.argv:
    sys.argv.remove(option_str)
    return True
  return False


if __name__ == '__main__':
  ext_module_list = []
  warnings_as_errors = '--warnings_as_errors'
  if get_option_from_sys_argv('--cpp_implementation'):
```

**Comment by dlan17** [comment]
@Arfrever hi, ping, could you update this? to address the reviews

File: dev-libs/protobuf/protobuf-3.19.4.ebuild
Comment (Arfrever) [reply dev-libs/protobuf/protobuf-3.19.4.ebuild:92]: A sample of ebuilds with arrays without unnecessary spaces, e.g. in `media-*/*/*.ebuild`:
(None of them are maintained or written by me)
```
$ grep '=([^ )]' media-*/*/*.ebuild
media-gfx/cura/cura-4.12.1.ebuild:DOCS=(README.md)
media-gfx/cura/cura-4.9.1.ebuild:DOCS=(README.md)
media-gfx/hydrus/hydrus-467.ebuild:     local DOCS=(COPYING README.md Readme.txt help_my_client_will_not_boot.txt db/)
media-gfx/hydrus/hydrus-467.ebuild:     local HTML_DOCS=("${S}"/help/)
media-gfx/hydrus/hydrus-468-r1.ebuild:  local DOCS=(COPYING README.md Readme.txt help_my_client_will_not_boot.txt db/)
media-gfx/hydrus/hydrus-468-r1.ebuild:  local HTML_DOCS=("${S}"/help/)
media-gfx/hydrus/hydrus-469.ebuild:     local DOCS=(COPYING README.md Readme.txt help_my_client_will_not_boot.txt db/)
media-gfx/hydrus/hydrus-469.ebuild:     local HTML_DOCS=("${S}"/help/)
media-gfx/hydrus/hydrus-471.ebuild:     local DOCS=(COPYING README.md Readme.txt help_my_client_will_not_boot.txt db/)
media-gfx/hydrus/hydrus-471.ebuild:     local HTML_DOCS=("${S}"/help/)
media-gfx/hydrus/hydrus-9999.ebuild:    local DOCS=(COPYING README.md Readme.txt help_my_client_will_not_boot.txt db/)
media-gfx/hydrus/hydrus-9999.ebuild:    local HTML_DOCS=("${S}"/help/)
media-gfx/jpeg2ps/jpeg2ps-1.9-r2.ebuild:PATCHES=("${FILESDIR}"/${P}-include.diff)
media-gfx/solvespace/solvespace-3.0.ebuild:             mycmakeargs+=("-DENABLE_OPENMP=OFF")
media-gfx/solvespace/solvespace-3.0.ebuild:             mycmakeargs+=("-DENABLE_OPENMP=ON")
media-libs/SoQt/SoQt-1.6.0.ebuild:DOCS=(AUTHORS ChangeLog HACKING NEWS README)
media-libs/SoXt/SoXt-1.4.0.ebuild:DOCS=(AUTHORS ChangeLog HACKING NEWS README TODO BUGS.txt)
media-libs/hamlib/hamlib-4.3.1.ebuild:DOCS=(AUTHORS NEWS PLAN README README.betatester README.developer)
media-libs/hamlib/hamlib-4.4.ebuild:DOCS=(AUTHORS NEWS PLAN README README.betatester README.developer)
media-libs/imlib/imlib-1.9.15-r5.ebuild:PVP=(${PV//[-\._]/ })
media-libs/mesa/mesa-21.3.5.ebuild:     emesonargs+=(-Dplatforms=${platforms#,})
media-libs/mesa/mesa-21.3.5.ebuild:             emesonargs+=(-Dglvnd=true)
media-libs/mesa/mesa-21.3.5.ebuild:             emesonargs+=(-Dglvnd=false)
media-libs/mesa/mesa-21.3.5.ebuild:                     emesonargs+=($(meson_use d3d9 gallium-nine))
media-libs/mesa/mesa-21.3.5.ebuild:                     emesonargs+=(-Dgallium-nine=false)
media-libs/mesa/mesa-21.3.5.ebuild:                     emesonargs+=($(meson_feature vaapi gallium-va))
media-libs/mesa/mesa-21.3.5.ebuild:                     emesonargs+=(-Dgallium-va=disabled)
media-libs/mesa/mesa-21.3.5.ebuild:                     emesonargs+=($(meson_feature vdpau gallium-vdpau))
media-libs/mesa/mesa-21.3.5.ebuild:                     emesonargs+=(-Dgallium-vdpau=disabled)
media-libs/mesa/mesa-21.3.5.ebuild:                     emesonargs+=($(meson_feature xa gallium-xa))
media-libs/mesa/mesa-21.3.5.ebuild:                     emesonargs+=(-Dgallium-xa=disabled)
media-libs/mesa/mesa-21.3.5.ebuild:                     emesonargs+=($(meson_feature xvmc gallium-xvmc))
media-libs/mesa/mesa-21.3.5.ebuild:                     emesonargs+=(-Dgallium-xvmc=disabled)
media-libs/mesa/mesa-21.3.5.ebuild:     emesonargs+=(-Dvulkan-layers=${vulkan_layers#,})
media-libs/mesa/mesa-21.3.5.ebuild:             DRI_DRIVERS+=("$@")
media-libs/mesa/mesa-21.3.5.ebuild:             GALLIUM_DRIVERS+=("$@")
media-libs/mesa/mesa-21.3.5.ebuild:             VULKAN_DRIVERS+=("$@")
media-libs/mesa/mesa-21.3.6.ebuild:     emesonargs+=(-Dplatforms=${platforms#,})
media-libs/mesa/mesa-21.3.6.ebuild:             emesonargs+=(-Dglvnd=true)
media-libs/mesa/mesa-21.3.6.ebuild:             emesonargs+=(-Dglvnd=false)
media-libs/mesa/mesa-21.3.6.ebuild:                     emesonargs+=($(meson_use d3d9 gallium-nine))
media-libs/mesa/mesa-21.3.6.ebuild:                     emesonargs+=(-Dgallium-nine=false)
media-libs/mesa/mesa-21.3.6.ebuild:                     emesonargs+=($(meson_feature vaapi gallium-va))
media-libs/mesa/mesa-21.3.6.ebuild:                     emesonargs+=(-Dgallium-va=disabled)
media-libs/mesa/mesa-21.3.6.ebuild:                     emesonargs+=($(meson_feature vdpau gallium-vdpau))
media-libs/mesa/mesa-21.3.6.ebuild:                     emesonargs+=(-Dgallium-vdpau=disabled)
media-libs/mesa/mesa-21.3.6.ebuild:                     emesonargs+=($(meson_feature xa gallium-xa))
media-libs/mesa/mesa-21.3.6.ebuild:                     emesonargs+=(-Dgallium-xa=disabled)
media-libs/mesa/mesa-21.3.6.ebuild:                     emesonargs+=($(meson_feature xvmc gallium-xvmc))
media-libs/mesa/mesa-21.3.6.ebuild:                     emesonargs+=(-Dgallium-xvmc=disabled)
media-libs/mesa/mesa-21.3.6.ebuild:     emesonargs+=(-Dvulkan-layers=${vulkan_layers#,})
media-libs/mesa/mesa-21.3.6.ebuild:             DRI_DRIVERS+=("$@")
media-libs/mesa/mesa-21.3.6.ebuild:             GALLIUM_DRIVERS+=("$@")
media-libs/mesa/mesa-21.3.6.ebuild:             VULKAN_DRIVERS+=("$@")
media-libs/mesa/mesa-22.0.0_rc1.ebuild: emesonargs+=(-Dplatforms=${platforms#,})
media-libs/mesa/mesa-22.0.0_rc1.ebuild:         emesonargs+=($(meson_use d3d9 gallium-nine))
media-libs/mesa/mesa-22.0.0_rc1.ebuild:         emesonargs+=(-Dgallium-nine=false)
media-libs/mesa/mesa-22.0.0_rc1.ebuild:         emesonargs+=($(meson_feature vaapi gallium-va))
media-libs/mesa/mesa-22.0.0_rc1.ebuild:         emesonargs+=(-Dgallium-va=disabled)
media-libs/mesa/mesa-22.0.0_rc1.ebuild:         emesonargs+=($(meson_feature vdpau gallium-vdpau))
media-libs/mesa/mesa-22.0.0_rc1.ebuild:         emesonargs+=(-Dgallium-vdpau=disabled)
media-libs/mesa/mesa-22.0.0_rc1.ebuild:         emesonargs+=($(meson_feature xa gallium-xa))
media-libs/mesa/mesa-22.0.0_rc1.ebuild:         emesonargs+=(-Dgallium-xa=disabled)
media-libs/mesa/mesa-22.0.0_rc1.ebuild:         emesonargs+=($(meson_feature xvmc gallium-xvmc))
media-libs/mesa/mesa-22.0.0_rc1.ebuild:         emesonargs+=(-Dgallium-xvmc=disabled)
media-libs/mesa/mesa-22.0.0_rc1.ebuild: emesonargs+=(-Dvulkan-layers=${vulkan_layers#,})
media-libs/mesa/mesa-22.0.0_rc1.ebuild:         GALLIUM_DRIVERS+=("$@")
media-libs/mesa/mesa-22.0.0_rc1.ebuild:         VULKAN_DRIVERS+=("$@")
media-libs/mesa/mesa-22.0.0_rc2.ebuild: emesonargs+=(-Dplatforms=${platforms#,})
media-libs/mesa/mesa-22.0.0_rc2.ebuild:         emesonargs+=($(meson_use d3d9 gallium-nine))
media-libs/mesa/mesa-22.0.0_rc2.ebuild:         emesonargs+=(-Dgallium-nine=false)
media-libs/mesa/mesa-22.0.0_rc2.ebuild:         emesonargs+=($(meson_feature vaapi gallium-va))
media-libs/mesa/mesa-22.0.0_rc2.ebuild:         emesonargs+=(-Dgallium-va=disabled)
media-libs/mesa/mesa-22.0.0_rc2.ebuild:         emesonargs+=($(meson_feature vdpau gallium-vdpau))
media-libs/mesa/mesa-22.0.0_rc2.ebuild:         emesonargs+=(-Dgallium-vdpau=disabled)
media-libs/mesa/mesa-22.0.0_rc2.ebuild:         emesonargs+=($(meson_feature xa gallium-xa))
media-libs/mesa/mesa-22.0.0_rc2.ebuild:         emesonargs+=(-Dgallium-xa=disabled)
media-libs/mesa/mesa-22.0.0_rc2.ebuild:         emesonargs+=($(meson_feature xvmc gallium-xvmc))
media-libs/mesa/mesa-22.0.0_rc2.ebuild:         emesonargs+=(-Dgallium-xvmc=disabled)
media-libs/mesa/mesa-22.0.0_rc2.ebuild: emesonargs+=(-Dvulkan-layers=${vulkan_layers#,})
media-libs/mesa/mesa-22.0.0_rc2.ebuild:         GALLIUM_DRIVERS+=("$@")
media-libs/mesa/mesa-22.0.0_rc2.ebuild:         VULKAN_DRIVERS+=("$@")
media-libs/mesa/mesa-9999.ebuild:       emesonargs+=(-Dplatforms=${platforms#,})
media-libs/mesa/mesa-9999.ebuild:               emesonargs+=($(meson_use d3d9 gallium-nine))
media-libs/mesa/mesa-9999.ebuild:               emesonargs+=(-Dgallium-nine=false)
media-libs/mesa/mesa-9999.ebuild:               emesonargs+=($(meson_feature vaapi gallium-va))
media-libs/mesa/mesa-9999.ebuild:               emesonargs+=(-Dgallium-va=disabled)
media-libs/mesa/mesa-9999.ebuild:               emesonargs+=($(meson_feature vdpau gallium-vdpau))
media-libs/mesa/mesa-9999.ebuild:               emesonargs+=(-Dgallium-vdpau=disabled)
media-libs/mesa/mesa-9999.ebuild:               emesonargs+=($(meson_feature xa gallium-xa))
media-libs/mesa/mesa-9999.ebuild:               emesonargs+=(-Dgallium-xa=disabled)
media-libs/mesa/mesa-9999.ebuild:               emesonargs+=($(meson_feature xvmc gallium-xvmc))
media-libs/mesa/mesa-9999.ebuild:               emesonargs+=(-Dgallium-xvmc=disabled)
media-libs/mesa/mesa-9999.ebuild:       emesonargs+=(-Dvulkan-layers=${vulkan_layers#,})
media-libs/mesa/mesa-9999.ebuild:               GALLIUM_DRIVERS+=("$@")
media-libs/mesa/mesa-9999.ebuild:               VULKAN_DRIVERS+=("$@")
media-libs/openimageio/openimageio-2.2.18.0-r1.ebuild:          use "${cpufeature%:*}" && mysimd+=("${cpufeature#*:}")
media-libs/openimageio/openimageio-2.2.18.0-r1.ebuild:  [[ -z ${mysimd} ]] && mysimd=("0")
media-libs/openimageio/openimageio-2.2.19.0.ebuild:             use "${cpufeature%:*}" && mysimd+=("${cpufeature#*:}")
media-libs/openimageio/openimageio-2.2.19.0.ebuild:     [[ -z ${mysimd} ]] && mysimd=("0")
media-libs/openimageio/openimageio-2.3.11.0-r2.ebuild:          use "${cpufeature%:*}" && mysimd+=("${cpufeature#*:}")
media-libs/openimageio/openimageio-2.3.11.0-r2.ebuild:  [[ -z ${mysimd} ]] && mysimd=("0")
media-libs/openimageio/openimageio-2.3.11.0.ebuild:             use "${cpufeature%:*}" && mysimd+=("${cpufeature#*:}")
media-libs/openimageio/openimageio-2.3.11.0.ebuild:     [[ -z ${mysimd} ]] && mysimd=("0")
media-libs/openimageio/openimageio-2.3.12.0-r2.ebuild:          use "${cpufeature%:*}" && mysimd+=("${cpufeature#*:}")
media-libs/openimageio/openimageio-2.3.12.0-r2.ebuild:  [[ -z ${mysimd} ]] && mysimd=("0")
media-libs/osl/osl-1.11.16.0-r2.ebuild:         use "${cpufeature%:*}" && mysimd+=("${cpufeature#*:}")
media-libs/osl/osl-1.11.16.0-r2.ebuild: [[ -z ${mysimd} ]] && mysimd=("0")
media-libs/osl/osl-1.11.17.0-r2.ebuild:         use "${cpufeature%:*}" && mysimd+=("${cpufeature#*:}")
media-libs/osl/osl-1.11.17.0-r2.ebuild: [[ -z ${mysimd} ]] && mysimd=("0")
media-libs/osl/osl-1.12.4.1-r1.ebuild:          use "${cpufeature%:*}" && mysimd+=("${cpufeature#*:}")
media-libs/osl/osl-1.12.4.1-r1.ebuild:  [[ -z ${mysimd} ]] && mysimd=("0")
media-libs/portmidi/portmidi-234.ebuild:                mycmakeargs+=(-DJAR_INSTALL_DIR="${EPREFIX}/usr/share/${PN}/lib")
media-libs/quarter/quarter-1.1.0.ebuild:DOCS=(AUTHORS ChangeLog NEWS README)
media-libs/simage/simage-1.8.0-r1.ebuild:DOCS=(AUTHORS ChangeLog NEWS README)
media-libs/simage/simage-1.8.1.ebuild:DOCS=(AUTHORS ChangeLog NEWS README)
media-libs/simage/simage-9999.ebuild:DOCS=(AUTHORS ChangeLog NEWS README)
media-libs/x264/x264-0.0.20190903-r1.ebuild:PATCHES=("${FILESDIR}"/${P}-STRINGS.patch)
media-libs/zimg/zimg-2.9.2.ebuild:PATCHES=("${FILESDIR}"/${PN}-2.9.2-gcc-10.patch)
media-plugins/grilo-plugins/grilo-plugins-0.3.14.ebuild:                emesonargs+=($(meson_feature gnome-online-accounts goa))
media-plugins/grilo-plugins/grilo-plugins-0.3.14.ebuild:                emesonargs+=(-Dgoa=disabled)
media-plugins/vdr-cinebars/vdr-cinebars-0.0.5.ebuild:PATCHES=("${FILESDIR}/${P}_makefile.diff")
media-plugins/vdr-mp3ng/vdr-mp3ng-0.0.1_pre5-r5.ebuild:DOCS=(HISTORY MANUAL README README-MORONIMO examples/network.sh.example)
media-plugins/vdr-weatherng/vdr-weatherng-0.0.8_pre3-r1.ebuild:PATCHES=("${FILESDIR}/${P}-i18n-fix.diff")
media-radio/flrig/flrig-1.4.2.ebuild:DOCS=(AUTHORS ChangeLog README)
media-radio/flrig/flrig-1.4.3.ebuild:DOCS=(AUTHORS ChangeLog README)
media-radio/flrig/flrig-1.4.4.ebuild:DOCS=(AUTHORS ChangeLog README)
media-sound/bpmdetect/bpmdetect-0.7.0_pre20170810.ebuild:PATCHES=("${FILESDIR}"/${P}-gcc-11.patch)
media-sound/cadence/cadence-0.9.1.ebuild:       myemakeargs=(PREFIX="${EPREFIX}/usr"
media-sound/cadence/cadence-9999.ebuild:        myemakeargs=(PREFIX="${EPREFIX}/usr"
media-sound/puddletag/puddletag-2.0.1-r1.ebuild:DOCS=(changelog NEWS THANKS TODO)
media-sound/puddletag/puddletag-2.1.0.ebuild:DOCS=(changelog NEWS THANKS TODO)
media-sound/puddletag/puddletag-2.1.1.ebuild:DOCS=(changelog NEWS THANKS TODO)
media-tv/mythtv/mythtv-31.0-r6.ebuild:  myconf+=(--prefix="${EPREFIX}"/usr)
media-tv/mythtv/mythtv-31.0-r6.ebuild:  myconf+=(--libdir="${EPREFIX}"/usr/$(get_libdir))
media-tv/mythtv/mythtv-31.0-r6.ebuild:  myconf+=(--libdir-name=$(get_libdir))
media-tv/mythtv/mythtv-31.0-r6.ebuild:  myconf+=(--mandir="${EPREFIX}"/usr/share/man)
media-tv/mythtv/mythtv-31.0-r6.ebuild:          myconf+=(--compile-type=debug)
media-tv/mythtv/mythtv-31.0-r6.ebuild:          myconf+=(--disable-stripping) # FIXME: does not disable for all files, only for some
media-tv/mythtv/mythtv-31.0-r6.ebuild:          myconf+=(--enable-valgrind) # disables timeouts for valgrind memory debugging
media-tv/mythtv/mythtv-31.0-r6.ebuild:          myconf+=(--compile-type=release)
media-tv/mythtv/mythtv-31.0-r6.ebuild:  has ccache "${FEATURES}" || myconf+=(--disable-ccache)
media-tv/mythtv/mythtv-31.0-r6.ebuild:  has distcc "${FEATURES}" || myconf+=(--disable-distcc)
media-tv/mythtv/mythtv-31.0-r6.ebuild:          myconf+=(--cpu="${i}")
media-tv/mythtv/mythtv-31.0-r6.ebuild:  myconf+=($(use_enable altivec))
media-tv/mythtv/mythtv-31.0-r6.ebuild:          myconf+=(--with-bindings=perl,python)
media-tv/mythtv/mythtv-31.0-r6.ebuild:          myconf+=(--without-bindings=python)
media-tv/mythtv/mythtv-31.0-r6.ebuild:          myconf+=(--with-bindings=perl)
media-tv/mythtv/mythtv-31.0-r6.ebuild:          myconf+=(--without-bindings=perl)
media-tv/mythtv/mythtv-31.0-r6.ebuild:          myconf+=(--with-bindings=python)
media-tv/mythtv/mythtv-31.0-r6.ebuild:          myconf+=(--without-bindings=perl,python)
media-tv/mythtv/mythtv-31.0-r6.ebuild:  use python && myconf+=(--python="${EPYTHON}")
media-tv/mythtv/mythtv-31.0-r6.ebuild:  myconf+=($(use_enable java bdjava))
media-tv/mythtv/mythtv-31.0-r6.ebuild:  myconf+=(--enable-symbol-visibility)
media-tv/mythtv/mythtv-31.0-r6.ebuild:  myconf+=(--enable-pic)
media-tv/mythtv/mythtv-31.0-r6.ebuild:          myconf+=(--enable-cross-compile --arch=$(tc-arch-kernel))
media-tv/mythtv/mythtv-31.0-r6.ebuild:          myconf+=(--cross-prefix="${CHOST}"-)
media-tv/mythtv/mythtv-31.0-r6.ebuild:  myconf+=($(use_enable bluray libbluray_external))
media-tv/mythtv/mythtv-31.0_p20210731-r1.ebuild:        myconf+=(--prefix="${EPREFIX}"/usr)
media-tv/mythtv/mythtv-31.0_p20210731-r1.ebuild:        myconf+=(--libdir="${EPREFIX}"/usr/$(get_libdir))
media-tv/mythtv/mythtv-31.0_p20210731-r1.ebuild:        myconf+=(--libdir-name=$(get_libdir))
media-tv/mythtv/mythtv-31.0_p20210731-r1.ebuild:        myconf+=(--mandir="${EPREFIX}"/usr/share/man)
media-tv/mythtv/mythtv-31.0_p20210731-r1.ebuild:                myconf+=(--compile-type=debug)
media-tv/mythtv/mythtv-31.0_p20210731-r1.ebuild:                myconf+=(--disable-stripping) # FIXME: does not disable for all files, only for some
media-tv/mythtv/mythtv-31.0_p20210731-r1.ebuild:                myconf+=(--enable-valgrind) # disables timeouts for valgrind memory debugging
media-tv/mythtv/mythtv-31.0_p20210731-r1.ebuild:                myconf+=(--compile-type=release)
media-tv/mythtv/mythtv-31.0_p20210731-r1.ebuild:        has ccache "${FEATURES}" || myconf+=(--disable-ccache)
media-tv/mythtv/mythtv-31.0_p20210731-r1.ebuild:        has distcc "${FEATURES}" || myconf+=(--disable-distcc)
media-tv/mythtv/mythtv-31.0_p20210731-r1.ebuild:                myconf+=(--cpu="${i}")
media-tv/mythtv/mythtv-31.0_p20210731-r1.ebuild:        myconf+=($(use_enable altivec))
media-tv/mythtv/mythtv-31.0_p20210731-r1.ebuild:                myconf+=(--with-bindings=perl,python)
media-tv/mythtv/mythtv-31.0_p20210731-r1.ebuild:                myconf+=(--without-bindings=python)
media-tv/mythtv/mythtv-31.0_p20210731-r1.ebuild:                myconf+=(--with-bindings=perl)
media-tv/mythtv/mythtv-31.0_p20210731-r1.ebuild:                myconf+=(--without-bindings=perl)
media-tv/mythtv/mythtv-31.0_p20210731-r1.ebuild:                myconf+=(--with-bindings=python)
media-tv/mythtv/mythtv-31.0_p20210731-r1.ebuild:                myconf+=(--without-bindings=perl,python)
media-tv/mythtv/mythtv-31.0_p20210731-r1.ebuild:        use python && myconf+=(--python="${EPYTHON}")
media-tv/mythtv/mythtv-31.0_p20210731-r1.ebuild:        myconf+=($(use_enable java bdjava))
media-tv/mythtv/mythtv-31.0_p20210731-r1.ebuild:        myconf+=(--enable-symbol-visibility)
media-tv/mythtv/mythtv-31.0_p20210731-r1.ebuild:        myconf+=(--enable-pic)
media-tv/mythtv/mythtv-31.0_p20210731-r1.ebuild:                myconf+=(--enable-cross-compile --arch=$(tc-arch-kernel))
media-tv/mythtv/mythtv-31.0_p20210731-r1.ebuild:                myconf+=(--cross-prefix="${CHOST}"-)
media-tv/mythtv/mythtv-31.0_p20210731-r1.ebuild:        myconf+=($(use_enable bluray libbluray_external))
media-video/atomicparsley-wez/atomicparsley-wez-0.9.6.ebuild:DOCS=(Changes.txt CREDITS)
media-video/lsdvd/lsdvd-0.17.ebuild:DOCS=(AUTHORS README ChangeLog)
media-video/mpv/mpv-0.33.1-r2.ebuild:   [[ ${PV} != *9999* ]] && mywafargs+=(--disable-build-date)
media-video/mpv/mpv-0.34.0.ebuild:      [[ ${PV} != *9999* ]] && mywafargs+=(--disable-build-date)
media-video/mpv/mpv-9999.ebuild:        [[ ${PV} != *9999* ]] && mywafargs+=(--disable-build-date)
media-video/shotcut/shotcut-21.10.31.ebuild:    use debug || myqmakeargs+=(DEFINES+=NDEBUG)
media-video/shotcut/shotcut-21.12.24.ebuild:    use debug || myqmakeargs+=(DEFINES+=NDEBUG)
media-video/shotcut/shotcut-9999.ebuild:        use debug || myqmakeargs+=(DEFINES+=NDEBUG)
media-video/vcdimager/vcdimager-2.0.1.ebuild:PATCHES=("${FILESDIR}/${P}-pkg-config.patch")
```
This contains e.g. ebuilds such as `media-libs/mesa` which were rewritten for switching from Autotools to Meson and cannot be accused of using old style or any such thing.

Comment (SoapGentoo) [reply dev-libs/protobuf/protobuf-3.19.4.ebuild:90]: Again, I make the final call here. Either you change it or this doesn't go in.

**Comment by SoapGentoo** [comment]
Ok, giving you another week before I close this PR

**Pull request closed without merging**
